docs/overview.md

```diff
@@ -168,31 +168,31 @@ The results are returned as a list of dictionaries, which can be easily converte
 Mesa now uses a new browser-based visualization system called SolaraViz. This allows for interactive, customizable visualizations of your models. Here's a basic example of how to set up a visualization:
 
 ```python
-from mesa.visualization import SolaraViz, make_space_component, make_plot_measure
+from mesa.visualization import SolaraViz, make_space_component, make_plot_component
 
 
 def agent_portrayal(agent):
-    return {"color": "blue", "size": 50}
+   return {"color": "blue", "size": 50}
 
 
 model_params = {
-    "N": {
-        "type": "SliderInt",
-        "value": 50,
-        "label": "Number of agents:",
-        "min": 10,
-        "max": 100,
-        "step": 1,
-    }
+   "N": {
+      "type": "SliderInt",
```

Comment:
What’s happening here? Moving to 3 space intends?

Reply:
I guess an artifact of the automatic renaming?

Reply:
Looks almost like some changed IDE setting. Strange that ruff likes it?

Reply:
It makes no sense and I did not touch that file directly but only indirectly by using pycharms refactor --> rename

Reply:
I have added the spaces back in to ensure the code is rendered correctly.